djangocms_admin_style/tests/integration/helpers/cms.js

```diff
@@ -0,0 +1,607 @@
+/* global document, localStorage */
+'use strict';
+
+module.exports = function (casperjs, settings) {
```

Comment:
just as a reminder - these helpers will only work in develop (some stuff is different from 3.2.x)
also we will need to find a way to publish it somehow